packages/react-reconciler/src/__tests__/ReactIncremental-test.internal.js

```diff
@@ -1916,8 +1916,7 @@ describe('ReactIncremental', () => {
       'Legacy context API has been detected within a strict-mode tree.\n\n' +
         'The old API will be supported in all 16.x releases, but applications ' +
         'using it should migrate to the new version.\n\n' +
-        'Please update the following components: Intl, ShowBoth, ShowLocale',
-      {withoutStack: true},
+        'Please update the following components: Intl, ShowLocale, ShowBoth',
```

Comment:
Should any of these tests verify that the stack is something reasonable (that it belongs to one of those components)?

Reply:
I don't follow? The shown stack (there's only one) should always be the first component in the comma separated list.

Reply:
My point is that none of these tests actually verify that. They verify that there is *a stack* but not what's in it.

Reply:
There is a test that does, I belive it's in ReactStrictMode-test.js

Reply:
Ah, ok. Fair enough.